### ISSUE TITLE
[Brave News] Content items with no image are missing from the feed

### ISSUE THREAD

**Issue by mattmcalister** [brave/brave-browser#27007]
Sources such as FT, Zerohedge, France24, Cointelegraph, The Kitchn, and WashingtonPost often have no image and, as a result, do not appear in the feed at all. 

Items with no images should be ignored by the image cards in the feed sequence, but they have a place to exist in the list cards -- the category, publisher and random list cards in the sequence.

**Comment by LorenzoMinto** [comment]
@fallaciousreasoning @aurangzaib048 this should be fixed right?

**Comment by aurangzaib048** [comment]
Yeah As we previously discussed, articles without images should still be included in the feed. However, it's important to note that they shouldn't be presented as Hero cards.

**Comment by LorenzoMinto** [comment]
@fallaciousreasoning you can close if desktop is aligned with what @aurangzaib048 said.
